drone_talk.py

```diff
@@ -0,0 +1,34 @@
+from discord.ext import commands
+from discord.utils import get
+import discord
+import messages
+from roles import HIVE_MXTRESS, DRONE
+from channels import COMMUNICATION_CHANNEL
+import re
+
+def get_user_id(username):
+	drone_id = re.search(r"\d{4}", username).group()
+	return drone_id if drone_id is not None else 0000;
+
+def has_role(member: discord.Member, role: str) -> bool:
+	return get(member.roles, name=role) is not None
+
+class Drone_Talk(commands.Cog):
```

Comment:
Maybe add some documentation that says, what this Cog does. Protip: Documentation for classes and functions is usually written like this (ttps://www.python.org/dev/peps/pep-0257/):
```suggestion
class Drone_Talk(commands.Cog):
	"""
	Awesome documentation, that can be displayed in IDEs.
	"""
```